overrides/android-override.json

```diff
@@ -565,6 +565,32 @@
         "clientContentFeatures": {
             "state": "enabled",
             "exceptions": [ ]
+        },
+        "mediaPlaybackRequiresUserGesture": {
+            "state": "enabled",
+            "exceptions": [
+                {
+                    "domain": "clearmemorysolution.com"
+                },
+                {
+                    "domain": "id.onlyfans.com"
+                },
+                {
+                    "domain": "naturemetabolism.com"
+                },
+                {
+                    "domain": "upperbodystretch.com"
+                },
+                {
+                    "domain": "app.trunarrative.cloud"
+                },
+                {
+                    "domain": "cloudapi.trustid.co.uk"
+                },
+                {
+                    "domain": "facialwebcapture.cloud.itau.com.br"
```

Comment:
Added 3 domains. See https://app.asana.com/0/0/1206420802766788/f and https://app.asana.com/0/0/1206420802766788/1206495276207661/f.

Reply:
Added 2 domains. See https://app.asana.com/0/0/1206420802766788/1206497348822710/f and https://app.asana.com/0/0/1206420802766788/1206500790260328/f.